challenge.rb

```diff
@@ -1,26 +1,49 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+	input.map do |s|
+		s.capitalize
+	end
 end
```

Comment:
Overall, LGTM. However, since this is such a short mapping, it would probably look nicer to make that a one liner using `{ }` curly braces to denote the block. Also, consider: would input.map(&:capitalize) work? Why?